package.json

```diff
@@ -10,12 +10,7 @@
   "license": "Apache-2.0",
   "author": "labforward GmbH",
   "type": "module",
-  "exports": {
-    ".": {
-      "require": null,
-      "default": "./dist/index.js"
-    }
-  },
+  "exports": "./dist/index.js",
```

Comment:
Couple questions: Why did we chose to _prevent_ `require()` imports? How is this now behaving differently for ESM import statements? IIUC `"default"` is the fallback for the main entry point `"."`, so I would have expected this to behave the exact same way